just/bluefin-apps.just

```diff
@@ -120,3 +120,127 @@ install-incus:
         echo "Run \"just devmode\" to turn on Developer mode."
         exit
     fi
+# Install Google Chrome # I have a desire to make a remark here.
+install-google-chrome:
+    #!/usr/bin/env sh
```

Comment:
It would be far preferable to have these as called scripts instead of inlining them inside the justfile.

This is advantageous since justfiles have strict formatting requirements which can break things.